packages/opentelemetry-core/src/platform/node/ShutdownNotifier.ts

```diff
@@ -13,15 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-export function afterGlobalShutdown(): void {
-  process.removeAllListeners('SIGTERM');
-}
 
-export function onGlobalShutdown(cb: () => void): void {
+export function handleGlobalShutdown(cb: () => void) {
   process.once('SIGTERM', cb);
+  return function removeCallbackFromGlobalShutdown(removecb: () => void) {
```

Comment:
remove `removecb` it is not needed

Reply:
I am having some trouble understanding why a developer would not want a separate callback when removing the listener?

Reply:
because you are getting already a function which does that, that's the whole point of that. You return function that does all cleanup for you without any more dependency.

Reply:
Got it, I changed this.